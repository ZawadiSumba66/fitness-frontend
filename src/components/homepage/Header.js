import { Link } from '@reach/router';
import { Navbar } from 'react-bootstrap';

const Header = () => (
  <Navbar
    className="d-flex justify-content-between py-3 align-items-center"
  >
    <div>
      <Link
        to="/"
        className="text-lg text-decoration-none font-weight-bolder text-light"
      >
        <i className="fas fa-heartbeat" />
        <span>KEEP IT FIT</span>
      </Link>
    </div>
    <div className="">
      <Link
        to="/login"
        className="px-4 text-decoration-none font-weight-bolder text-uppercase text-dark text-sm-light"
      >
        Sign in
      </Link>
      <Link to="/signup" className="header-btn font-weight-bolder text-uppercase text-light py-2 px-3 button-nav btn-dark">
        Sign up
      </Link>
    </div>
  </Navbar>
);
export default Header;
