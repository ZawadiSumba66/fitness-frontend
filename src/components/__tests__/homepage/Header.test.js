import renderer from 'react-test-renderer';
import Header from '../../homepage/Header';

it('matches the header snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
