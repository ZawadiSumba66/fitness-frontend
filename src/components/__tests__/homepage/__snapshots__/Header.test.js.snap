// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the header snapshot 1`] = `
<nav
  className="d-flex justify-content-between py-3 align-items-center navbar navbar-expand navbar-light"
>
  <div>
    <a
      aria-current="page"
      className="text-lg text-decoration-none font-weight-bolder text-light"
      href="/"
      onClick={[Function]}
    >
      <i
        className="fas fa-heartbeat"
      />
      <span>
        KEEP IT FIT
      </span>
    </a>
  </div>
  <div
    className=""
  >
    <a
      className="px-4 text-decoration-none font-weight-bolder text-uppercase text-dark"
      href="/login"
      onClick={[Function]}
    >
      Sign in
    </a>
    <a
      className="header-btn font-weight-bolder text-uppercase text-light py-2 px-3 button-nav btn-dark"
      href="/signup"
      onClick={[Function]}
    >
      Sign up
    </a>
  </div>
</nav>
`;
